atc/db/team.go

```diff
@@ -788,6 +801,54 @@ func (t *team) FindCheckContainers(logger lager.Logger, pipelineName string, res
 	return containers, checkContainersExpiresAt, nil
 }
 
+type UpdateName struct {
+	OldName  string
+	NewName  string
+	TempName int
+}
+
+func (t *team) updateName(tx Tx, jobs []atc.JobConfig, pipelineID int) error {
+	jobsToUpdate := []UpdateName{}
+	counter := 0
+
+	for _, job := range jobs {
```

Comment:
this could be `for counter, job := range jobs {` to avoid the extra var

Reply:
Oh good catch!